### DISCUSSION TITLE
Tag Evolution (Add a Badge)

### DISCUSSION THREAD

**Discussion by LucasAlbuquerque0** [Telefonica/mistica-design#1572] [Proposals]
### Goal

Having the Badge functionality within the Tag component. Opens the status indication within the tag, actions that need to be taken or notifications.

### Context

Within the "Requests" flow, Badges within the tags are extremely necessary. Making it easier for the end user to have a status on the actions they are expecting or can take. For example, when any type of service is requested, some information may be requested or even have new developments within the request statuses. With the badge, we indicate whether there are actions within the card that the user needs and whether there are new updates. This is one of the flows that will benefit from the use of badges within Tags.

### Impact

Improve the way the user interacts with actions within the app. Making it easier to know when actions are necessary, without relying solely on the device's notifications field, being able to update itself natively.

### Discussion Questions

Is there some level of complexity that might prevent addition? Or specific rule?

### Do you have a visual proposal?

https://www.figma.com/file/wvqQ83vwmg98TeE3Vki6E8/Tag-com-badge?type=design&node-id=504%3A8248&mode=design&t=xuZ8j9yH6Mnb5iz7-1

**Comment by AnaMontes11** [comment]
Hi! @LucasAlbuquerque0 
We have reviewed your proposal and think it could be a good evolution for the tag to cover this type of cases. 
Thank you for your contribution!
